app/presenters/pagination_presenter.rb

```diff
@@ -0,0 +1,78 @@
+class PaginationPresenter
+  include Rails.application.routes.url_helpers
+
+  attr_reader :resource
+
+  def initialize(resource)
+    @resource= resource
+  end
+
+  def meta
+    {
+      total: total_resources,
+      total_pages: total_pages,
+      current_page: current_page,
+      is_first_page: is_first_page?,
+      is_last_page: is_last_page?,
+      previous_page: previous_page,
+      next_page: next_page,
+      out_of_bounds: out_of_bounds?,
+      # next_offset: offset,
+    }
+  end
+
+  private
+    def total_resources
+      resource.total_entries
+    end
+
+    def current_page
+      resource.current_page
+    end
+
+    def total_pages
+      resource.total_pages
+    end
+
+    def is_first_page?
```

Comment:
Predicate methods do not need to start with `is_?`. Since there is a question mark, we know that it returns a boolean value. So this method could be refactored to `first_page?`